client/src/components/Workflow/WorkflowDropdown.vue

```diff
@@ -45,6 +51,13 @@
                 <span class="fa fa-eye fa-fw mr-1" />
                 <span>View</span>
             </a>
+            <a
+                v-if="workflow.source_metadata && workflow.source_metadata.trs_server == 'dockstore'"
+                class="dropdown-item"
+                :href="urlDockstore">
+                <span class="fa fa-cube fa-fw mr-1" />
+                <span>View on Dockstore</span>
```

Comment:
```suggestion
                <span>{{trsLabel}}</span>
```
and then create a trsLabel computed property